ingestion/src/metadata/ingestion/source/database/mysql/metadata.py

```diff
@@ -39,6 +40,14 @@ class MysqlSource(CommonDbSourceService):
     Database metadata from Mysql Source
     """
 
+    def __init__(self, config: WorkflowSource, metadata: OpenMetadata):
+        self.ssl_manager = None
```

Comment:
This code appears to be duplicated several times. I suggest putting in a function or a mixin.

Reply:
Taken care of the same, Thanks 🙌